tests/console/sshd.pm

```diff
@@ -121,7 +121,13 @@ sub run {
     assert_script_run 'until ss -tulpn|grep sshd|egrep "4242|5252";do sleep 1;done';
 
     # Scan public keys on forwarded ports
-    assert_script_run "ssh-keyscan -p 4242 localhost >> ~/.ssh/known_hosts";
+    # Add a workaround about bsc#1193275 in FIPS test
+    my $output = script_output("ssh-keyscan -p 4242 localhost >> ~/.ssh/known_hosts", proceed_on_failure => 1);
+    if ($output =~ /choose_kex: unsupported KEX method curve25519-sha256/) {
+        record_soft_failure('bsc#1193275', "Currently the curve25519 is not FIPS approved");
+        return;
```

Comment:
what does the `return` use for? exit the test?

Reply:
hmm, I think it is not necessary, right?

Reply:
If you want to the following tests continue run, it is not necessary.

Reply:
good comment, I think I should remove it, thanks.

Reply:
DONE.